packages/theme/src/lib/legacy-theme.ts

```diff
@@ -8,15 +8,24 @@ const containedButtonStyles = (color: string) => ({
   '&:hover': {
     backgroundColor: tokens[`color${color}Dark` as keyof typeof tokens],
     boxShadow: 'none',
+    color: color === 'Warning' && tokens.colorTextInverse,
   },
   '&:focus': {
     backgroundColor: tokens[`color${color}Dark` as keyof typeof tokens],
     outline: '2px solid white',
     boxShadow: `0 0 0px 4px ${tokens[`color${color}Dark` as keyof typeof tokens]}`,
+    color: color === 'Warning' && tokens.colorTextInverse,
   },
   '&:active': {
     backgroundColor: tokens[`color${color}Main` as keyof typeof tokens],
   },
+  '&.Mui-disabled': {
+    opacity: 0.65,
+  },
+  '&.Mui-selected': {
```

Comment:
I'm hesitant to base this off the reactstrap active colors for toggle button, specifically primary/secondary. The main place this is used is feedback and would make it inaccessible since we're not adding additional logic in the component for legacy theming. I think we should strive for a happy medium with the legacy theming of this component where people using the old tertiary can still swap over easy but doesn't hinder feedback.